src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/Extensions/ApplicationInsightsServiceCollectionExtensions.cs

```diff
@@ -80,6 +80,7 @@ public static IServiceCollection AddApplicationInsights(this IServiceCollection
             services.AddSingleton<ITelemetryInitializer, MetricSdkVersionTelemetryInitializer>();
             services.AddSingleton<QuickPulseInitializationScheduler>();
             services.AddSingleton<QuickPulseTelemetryModule>();
+            services.AddSingleton<ITelemetryModule, TransmissionStatusTelemetryModule>();
```

Comment:
This dependency is used in the cold start code path. Is that correct?

Reply:
yes